CHANGELOG.md

```diff
@@ -22,15 +22,27 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 ## New features
 
 * Traits for "metaprogramming" with `seqan3::type_list` and type packs have been added.
+* Support for PowerPC has been added.
 
 ## API changes
 
 ### The `type_list` header has moved
 
-If you included `<seqan3/core/type_list.hpp>` you need to change the path to `<seqan3/core/type_list/type_list.hpp>`.
+If you included `<seqan3/core/type_list.hpp>`, you need to change the path to `<seqan3/core/type_list/type_list.hpp>`.
+
+### The `seqan3::fm_index` and `seqan3::bi_fm_index` now use strong typed template parameters
+
+Instead of `false` and `true`, [seqan3::text_layout::single](@ref seqan3::text_layout) and
```

Comment:
```suggestion
Instead of `false` and `true`, [seqan3::text_layout::single](\ref seqan3::text_layout) and
```

How do these links look when people open the MD file from the directory? Isn't a link for generated automatically if you just write `seqan3::text_layout::single`?

Reply:
Either I do `\ref ...` or `[text](@ref link)` (sic).
It can't link to `::single` or `::collection`; so I have to manually link to the enum instead.